Adyen/Core/Components/InstantPaymentComponent.swift

```diff
@@ -56,14 +56,11 @@ public final class InstantPaymentComponent: PaymentComponent {
 
     /// Generate the payment details and invoke PaymentsComponentDelegate method.
     public func initiatePayment() {
-        sendTelemetryEvent()
+        // we can attempt to fetch the checkoutAttemptId but it won't be ready for the payment
```

Comment:
```suggestion
        // We are not attempting to fetch the checkoutAttemptId here as it won't be ready for the payment and we don't want to block the payment because of an analytics call
```

Or something like that to give it a bit more context